core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -108,7 +108,8 @@
         </div>
       </div>
 
-      <div ng-if="collectionSummary && recommendedExplorationSummaries.length === 0">
+      <div ng-if="collectionSummary && recommendedExplorationSummaries.
+      length === 0">
```

Comment:
Let's keep the two previous lines as one line -- we never break on "." in HTML. IT's OK if lines in HTML templates end up being a bit long, they can get hard to read otherwise.